ios/FluentUI/Divider/Divider.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public enum MSFDividerOrientation: Int {
+    case horizontal
+    case vertical
+}
+
+/// UIKit wrapper that exposes the SwiftUI Divider implementation.
+@objc open class MSFDivider: NSObject, FluentUIWindowProvider {
+
+    /// The UIView representing the Fluent Divider.
+    @objc open var view: UIView {
+        return hostingController.view
+    }
+
+    /// The object that groups properties that allow control over the Divider appearance.
+    @objc open var state: MSFDividerState {
+        return divider.state
+    }
+
+    /// Creates a new MSFDivider instance.
+    ///  - Parameters:
+    ///   - style: The DividerStyle value used by the Divider.
+    ///   - orientation: The DividerOrientation used by the Divider.
+    ///   - spacing: The DividerSpacing used by the Divider.
+    @objc public init(style: MSFDividerStyle = .default, orientation: MSFDividerOrientation = .horizontal, spacing: MSFDividerSpacing = .none) {
```

Comment:
There's no initializer variant that takes a `FluentUIStyle` for theming. Is this intentional?